luaui/Widgets_BAR/cmd_select_commander.lua

```diff
@@ -0,0 +1,32 @@
+function widget:GetInfo()
+  return {
+    name = "SelectCommander",
+    desc = "Press ctrl+c to select the commander",
+    author = "Teifion",
+    date = "26/10/2020",
+    license = "GNU GPL, v2 or later",
+    layer = 0,
+    enabled = true
+  }
+end
+
+function widget:Update()
+  local t = Spring.GetGameSeconds()
+
+  -- Spectators cannot select their commander, they don't have one
```

Comment:
Spectators can still select units though. What if they want to select the commander of the team they are spectating?

Also, what if a spectator joins a team (via `/team`)? If the widget gets disabled then the option gets turned off for the rest of the game.

Reply:
Those are both very good conditions I didn't think about, I'll get it changed.